### ISSUE TITLE
Disable Mouse OSD

### ISSUE THREAD

**Issue by zekedran** [Code52/carnac#183]
Show `mouse clicks` option under `Mouse` Tab is only disabling the circles that reveal where the pointer is currently. But is there a way to turn off Mouse action OSDs? or to make them really small?

![image](https://user-images.githubusercontent.com/8567893/38449377-41d11ecc-3a06-11e8-9c0e-cb9d92c376b4.png)

My screen is quickly getting filled up by those

**Comment by mleonelli** [comment]
+1 for this feature, it would be really useful to have mouse clicks but not mouse OSD

**Comment by dgtexan** [comment]
Yes, I would like the ability to disable mouse OSD while leaving mouseclicks circle and other keystrokes OSD enabled.

**Comment by Hezkore** [comment]
I've been searching for this feature.
I don't feel like displaying the mouse buttons is needed when there's already a circle indicating when you're pressing a mouse button.

**Comment by AndroidDeveloperLB** [comment]
How did you enable mouse clicks anyway?
I can't see it:

![image](https://user-images.githubusercontent.com/5357526/69129808-e4138980-0ab7-11ea-8321-440e76921fac.png)

**Comment by jwiens-27** [comment]
I know this thread is 2 years old, but the lack of this feature annoys me so much. Please implement this at some point

**Comment by abaumgaertner** [comment]
wrong fork? https://github.com/bfritscher/carnac/
